tests/test_app.py

```diff
@@ -0,0 +1,179 @@
+"""
+Tests for the App class
+"""
+
+from dbt_schema_builder.app import App
+from dbt_schema_builder.relation import Relation
+from dbt_schema_builder.schema import Schema
+
+
+def test_add_source_to_new_schema():
+    schema_1 = Schema('LMS_TEST_RAW', [], [])
+    schema_2 = Schema('LMS_RAW', [], [])
+    schema_3 = Schema('LMS_STITCH_RAW', [], [])
+    raw_schemas = [schema_1, schema_2, schema_3]
+    app = App(
+        raw_schemas, 'LMS', 'models/PROD/LMS', 'models/PROD/LMS/LMS.yml',
```

Comment:
These constructors are pretty complicated, can we put each param on its own line to at least simplify the view? Or did linting make you do this?

Reply:
can do